### PR TITLE
Chi distribution changes

### PR DESCRIPTION
**Pull request by maymoo99**
Split central/non-central chi distributions up.
Added comments
Added Chi distribution tranformations (F and T distributions)

### CODE REVIEW

File: stdlib/chiDistributions.hk [hunk 1]
```diff
@@ -1,44 +1,64 @@
...
-		(q[i]/stdevs[i])^2
+####################################
+# Chi distribution Transformations #
+####################################
 
-def nonCentralChi(means array(real), stdevs array(prob)):
-	q <~ nonCentralChiSq(means,stdevs)
-	return sqrt(real2prob(q))
+# F_distribution "arises frequently as the null distribution of a test statistic, most notably in the analysis of variance, e.g., F-test." - Wikipedia
+def F_distribution(n1 nat, n2 nat):
```
Comment (zaxtax) [inline stdlib/chiDistributions.hk]: I don't think's you need to say F-distribution. Just say F.

File: stdlib/chiDistributions.hk [hunk 2]
```diff
@@ -1,44 +1,64 @@
...
+# This function takes 2 arrays of size N, containing the means and standard deviations
+# of the N normal distributions.
 def chiSq(means array(real), stdevs array(prob) ):
-	#TODO: check sizes of means==stdevs
+	#TODO: error check sizes of means==stdevs
 	q <~ plate _ of size(means): normal(means[_],stdevs[_])
 	return summate i from 0 to size(q):
 		((q[i]-means[i])/stdevs[i])^2
 
+# The chi distribution. Takes 2 arrays of size N like above.
```
Comment (zaxtax) [inline stdlib/chiDistributions.hk]: Does anyone use this distribution?

File: stdlib/chiDistributions.hk [hunk 3]
```diff
@@ -1,44 +1,64 @@
...
+def nonCentralT(n nat, delta prob):
+	U <~ normal(0,1)
+	X <~ standardChi(n)
+	return (U + delta)*sqrt(n)/(X)
 
-def ncp(means array(real), stdevs array(prob)):
-	return nonCentralityParameter(means, stdevs)
+# T distribution approximates normal distribution when 
+# sample size is small and standard deviation is unknown
+def T_distribution(n nat):
```
Comment (zaxtax) [inline stdlib/chiDistributions.hk]: Drop the _distribution bit

**Comment by zaxtax** [comment]
Rebase and I think we good.

**Comment by maymoo99** [comment]
Rebased and resubmitted changes

**Pull request closed without merging**